File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    result = set()
    for i in range(0, len(numbers)-1): # 0, 1, 2, 3
        for j in range(len(numbers)-(i+1)): # 0(0, 1, 2, 3) / 1(0, 1, 2)
            result.add(numbers[i] + numbers[i+j+1])
    return sorted(result)
        

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
const solution = (numbers) => {
    const result_set = new Set();
    
    numbers.forEach((num, idx) => {
        for (let i=0; i<idx; i++) {
            result_set.add(num + numbers[i]);
        }
    });
    return [...result_set].sort((a, b) => a - b);
}